Pear/Cells/MessagesTableViewCell.swift

```diff
@@ -90,11 +90,18 @@ class MessagesTableViewCell: UITableViewCell {
     }
 
     func configure(for pair: MatchedUser, status: String, week: Int) {
+        let shouldDisplayCurrentPear = status == Constants.Match.canceled || status == Constants.Match.inactive
         profileImage.kf.setImage(with: URL(string: pair.profilePicUrl ?? ""))
         nameLabel.text = "\(pair.firstName) \(pair.lastName)"
-        infoImage.isHidden = (status == Constants.Match.canceled || status == Constants.Match.inactive) ? true : false
-        infoLabel.isHidden = (status == Constants.Match.canceled || status == Constants.Match.inactive) ? true : false
+        infoImage.isHidden = shouldDisplayCurrentPear ? true : false
```

Comment:
Pretty sure you can just leave this as `infoImage.isHidden = shouldDisplayCurrentPear`